lib/pfconfig/manager.pm

```diff
@@ -39,233 +38,304 @@ use List::MoreUtils qw(any firstval uniq);
 use Scalar::Util qw(refaddr reftype tainted blessed);
 use UNIVERSAL::require;
 use Data::Dumper;
-use pfconfig::backend::bdb;
+use pfconfig::backend::mysql;
```

Comment:
For now we don't and in the end, we'll support only MariaDB. The abstraction is there in case we need to change the implementation in an emergency. The factory would be overkill IMO since this is used once.